File: src/models/comentarioProd.java
package models;

import java.sql.Date;
import java.sql.Time;
import java.util.Calendar;

import service.dbProd;

public class comentarioProd {
	
	public comentarioProd(int entrada_id) {
		super();
		this.entrada_id = entrada_id;
		this.jefe1 = "";
		this.jefe2 = "";
		this.jefe3 = "";
		this.jefe4 = "";
		this.jefe5 = "";
	}
	
	public String getJefe1() {
		return jefe1;
	}

	public void setJefe1(String jefe1) {
		this.jefe1 = jefe1;
	}

	public String getJefe1fecha() {
		return jefe1fecha;
	}

	public void setJefe1fecha(String jefe1fecha) {
		this.jefe1fecha = jefe1fecha;
	}

	public String getJefe1hora() {
		return jefe1hora;
	}

	public void setJefe1hora(String jefe1hora) {
		this.jefe1hora = jefe1hora;
	}

	public String getJefe2() {
		return jefe2;
	}

	public void setJefe2(String jefe2) {
		this.jefe2 = jefe2;
	}

	public String getJefe2fecha() {
		return jefe2fecha;
	}

	public void setJefe2fecha(String jefe2fecha) {
		this.jefe2fecha = jefe2fecha;
	}

	public String getJefe2hora() {
		return jefe2hora;
	}

	public void setJefe2hora(String jefe2hora) {
		this.jefe2hora = jefe2hora;
	}

	public String getJefe3() {
		return jefe3;
	}

	public void setJefe3(String jefe3) {
		this.jefe3 = jefe3;
	}

	public String getJefe3fecha() {
		return jefe3fecha;
	}

	public void setJefe3fecha(String jefe3fecha) {
		this.jefe3fecha = jefe3fecha;
	}

	public String getJefe3hora() {
		return jefe3hora;
	}

	public void setJefe3hora(String jefe3hora) {
		this.jefe3hora = jefe3hora;
	}

	public String getJefe4() {
		return jefe4;
	}

	public void setJefe4(String jefe4) {
		this.jefe4 = jefe4;
	}

	public String getJefe4fecha() {
		return jefe4fecha;
	}

	public void setJefe4fecha(String jefe4fecha) {
		this.jefe4fecha = jefe4fecha;
	}

	public String getJefe4hora() {
		return jefe4hora;
	}

	public void setJefe4hora(String jefe4hora) {
		this.jefe4hora = jefe4hora;
	}

	public String getJefe5() {
		return jefe5;
	}

	public void setJefe5(String jefe5) {
		this.jefe5 = jefe5;
	}

	public String getJefe5fecha() {
		return jefe5fecha;
	}

	public void setJefe5fecha(String jefe5fecha) {
		this.jefe5fecha = jefe5fecha;
	}

	public String getJefe5hora() {
		return jefe5hora;
	}

	public void setJefe5hora(String jefe5hora) {
		this.jefe5hora = jefe5hora;
	}

	public int getEntrada_id() {
		return entrada_id;
	}

	public void setEntrada_id(int entrada_id) {
		this.entrada_id = entrada_id;
	}
	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}
	
	public String getTramitadoPor() {
		return tramitadoPor;
	}

	public void setTramitadoPor(String tramitadoPor) {
		this.tramitadoPor = tramitadoPor;
	}
	
	int id;
	String jefe1;
	String jefe1fecha = "";
	String jefe1hora;
	String jefe2;
	String jefe2fecha;
	String jefe2hora;
	String jefe3;
	String jefe3fecha;
	String jefe3hora;
	String jefe4;
	String jefe4fecha;
	String jefe4hora;
	String jefe5;
	String jefe5fecha;
	String jefe5hora;
	String tramitadoPor;
	

	int entrada_id;
	
	 public void save(){
	        dbProd con = new dbProd();
	        con.connect();
	        con.saveComentario(this);
	        con.close();
	    }
}

File: src/models/comentarioProd.js
class ComentarioProd {
    constructor(entrada_id) {
        this.id = null; // Or generate a unique ID
        this.entrada_id = entrada_id;
        this.jefe1 = "";
        this.jefe1fecha = "";
        this.jefe1hora = "";
        this.jefe2 = "";
        this.jefe2fecha = "";
        this.jefe2hora = "";
        this.jefe3 = "";
        this.jefe3fecha = "";
        this.jefe3hora = "";
        this.jefe4 = "";
        this.jefe4fecha = "";
        this.jefe4hora = "";
        this.jefe5 = "";
        this.jefe5fecha = "";
        this.jefe5hora = "";
        this.tramitadoPor = "";
    }

    // Getters
    getId() { return this.id; }
    getEntradaId() { return this.entrada_id; }
    getJefe1() { return this.jefe1; }
    getJefe1Fecha() { return this.jefe1fecha; }
    getJefe1Hora() { return this.jefe1hora; }
    getJefe2() { return this.jefe2; }
    getJefe2Fecha() { return this.jefe2fecha; }
    getJefe2Hora() { return this.jefe2hora; }
    getJefe3() { return this.jefe3; }
    getJefe3Fecha() { return this.jefe3fecha; }
    getJefe3Hora() { return this.jefe3hora; }
    getJefe4() { return this.jefe4; }
    getJefe4Fecha() { return this.jefe4fecha; }
    getJefe4Hora() { return this.jefe4hora; }
    getJefe5() { return this.jefe5; }
    getJefe5Fecha() { return this.jefe5fecha; }
    getJefe5Hora() { return this.jefe5hora; }
    getTramitadoPor() { return this.tramitadoPor; }

    // Setters
    setId(id) { this.id = id; }
    setEntradaId(entrada_id) { this.entrada_id = entrada_id; }
    setJefe1(jefe1) { this.jefe1 = jefe1; }
    setJefe1Fecha(jefe1fecha) { this.jefe1fecha = jefe1fecha; }
    setJefe1Hora(jefe1hora) { this.jefe1hora = jefe1hora; }
    setJefe2(jefe2) { this.jefe2 = jefe2; }
    setJefe2Fecha(jefe2fecha) { this.jefe2fecha = jefe2fecha; }
    setJefe2Hora(jefe2hora) { this.jefe2hora = jefe2hora; }
    setJefe3(jefe3) { this.jefe3 = jefe3; }
    setJefe3Fecha(jefe3fecha) { this.jefe3fecha = jefe3fecha; }
    setJefe3Hora(jefe3hora) { this.jefe3hora = jefe3hora; }
    setJefe4(jefe4) { this.jefe4 = jefe4; }
    setJefe4Fecha(jefe4fecha) { this.jefe4fecha = jefe4fecha; }
    setJefe4Hora(jefe4hora) { this.jefe4hora = jefe4hora; }
    setJefe5(jefe5) { this.jefe5 = jefe5; }
    setJefe5Fecha(jefe5fecha) { this.jefe5fecha = jefe5fecha; }
    setJefe5Hora(jefe5hora) { this.jefe5hora = jefe5hora; }
    setTramitadoPor(tramitadoPor) { this.tramitadoPor = tramitadoPor; }
}

module.exports = ComentarioProd;
